tests/RedisTest.php

```diff
@@ -3318,16 +3318,18 @@ public function testObject() {
         $this->redis->lpush('key', 'value');
 
         /* Newer versions of redis are going to encode lists as 'quicklists',
-         * so 'quicklist' or 'ziplist' is valid here */
+         * redis >= 7.2 as 'listpack'
+         * so 'quicklist' or 'ziplist' or 'listpack' are valid here */
         $str_encoding = $this->redis->object('encoding', 'key');
-        $this->assertTrue($str_encoding === "ziplist" || $str_encoding === 'quicklist');
+        $this->assertTrue($str_encoding === "ziplist" || $str_encoding === 'quicklist' || $str_encoding === 'listpack', $str_encoding);
```

Comment:
I suggest using `version_compare($this->version, "7.2")`

Reply:
in this case we need to know which version use "quicklist" and which one use "ziplist" ?

Reply:
done (btw, I still prefer previous simpler version)